content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md

```diff
@@ -0,0 +1,228 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Binding"
+content_type: "api_reference"
+description: "Binding 即将一个对象与另一个对象联系起来;例如，一个 pod 被调度程序绑定到一个节点。"
+title: "Binding"
+weight: 9
+auto_generated: true
+---
+
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+
+## Binding {#Binding}
+<!--
+Binding ties one object to another; for example, a pod is bound to a node by a scheduler. Deprecated in 1.7, please use the bindings subresource of pods instead.
+-->
+Binding 即将一个对象与另一个对象联系起来;例如，一个 pod 被调度程序绑定到一个节点。1.7已弃用，请使用 pods 的 bindings 子资源。
+<hr>
+
+- **apiVersion**: v1
+
+
+- **kind**: Binding
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>), required
+
+  The target object that you want to bind to the standard object.
+-->
+  标准对象的元数据 更多信息: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>), 必需的
+
+  要绑定到标准对象的目标对象。
+
+## Operations {#Operations}
+
+<hr>
+
+<!--
+### `create` create a Binding
+
+#### HTTP Request
+
+POST /api/v1/namespaces/{namespace}/bindings
+
+#### Parameters
+-->
+### `创建` 创建一个 Binding
```

Comment:
```suggestion
### `create` 创建一个 Binding
```

Reply:
Thanks.It has been modified according to comments.